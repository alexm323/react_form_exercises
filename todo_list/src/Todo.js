import React from 'react';

const Todo = ({ task, handleRemove, id }) => {
    const remove = () => handleRemove(id)
    return (
        <div>
            <h3>{task}</h3>
            <button onClick={remove}>Remove Task</button>
        </div>
    )
}

export default Todo;