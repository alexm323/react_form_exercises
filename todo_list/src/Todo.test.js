import React from 'react';
import { render } from '@testing-library/react';
import Todo from './Todo';

it('renders without crashing', () => {
    render(<Todo />);
});

it('creates a snapshot', () => {
    const { asFragment } = render(<Todo />);
    expect(asFragment()).toMatchSnapshot();
});
