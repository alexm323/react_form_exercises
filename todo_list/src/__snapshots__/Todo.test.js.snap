// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates a snapshot 1`] = `
<DocumentFragment>
  <div>
    <h3 />
    <button>
      Remove Task
    </button>
  </div>
</DocumentFragment>
`;
